pom.xml

```diff
@@ -39,6 +39,75 @@
     </repositories>
 
     <dependencies>
+		<dependency>
+			<groupId>org.osgi</groupId>
+			<artifactId>osgi.cmpn</artifactId>
+			<version>6.0.0</version>
+			<scope>provided</scope>
+		</dependency>
+		<dependency>
```

Comment:
please add right scopes for each of the dependencies.

same applies to manager plugin...maybe create an issue on the sample plugin so it can be tracked and fixed